requirements.txt

```diff
@@ -4,4 +4,4 @@ pandas
 numpy
 deap
 oemof
-scipy
\ No newline at end of file
+scipy
```

Comment:
Where is this imported?

Reply:
https://github.com/rl-institut/smooth/blob/dev/smooth/components/component_functions/all_component_functions.py